### ISSUE TITLE
Quick Actions & Import Suggestions Width is very small

### ISSUE THREAD

**Issue by SzymonDziak** [drcika/apc-extension#117]
Current:
![Screenshot 2023-12-21 at 2 37 34 AM](https://github.com/drcika/apc-extension/assets/7705226/be306b6e-aa6b-47c1-abe4-4904e6073cce)

Desired:
![Screenshot 2023-12-21 at 2 39 13 AM](https://github.com/drcika/apc-extension/assets/7705226/e642ff2e-1d28-4054-8ff3-44e46f223327)

I'm not quite sure how I can customize this. I'm currently using the default template settings provided by the APC-Extension creator.

**Comment by drcika** [comment]
you have vscode resizes
https://github.com/drcika/apc-extension/assets/35919606/38737e81-eaf8-4f7f-9746-7f08c9be1955

**Comment by SzymonDziak** [comment]
@drcika yes, you are right. But this doesn't work for the Quick Fix and Auto Imports. The resizes only work for autocomplete when writing code. I need to resize the imports screen.

**Comment by SzymonDziak** [comment]
Any thoughts?

**Comment by Camoukey** [comment]
bump
